sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -333,21 +333,6 @@ public void recognizeEntitiesForListWithOptions(HttpClient httpClient, TextAnaly
                 .verifyComplete());
     }
 
-    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
```

Comment:
Do you think just a TODO Comment with the issue link should help better rather than removing. It would be a better diff for review.

Reply:
Either way works. But it is more clean to delete them right now and create an issue to remain the changes.